### PR TITLE
Feature/create detail service

### PR DESCRIPTION
**Pull request by CBinh710**
https://github.com/users/CBinh710/projects/2/views/1?pane=issue&itemId=72043822
image

### CODE REVIEW

File: practice-one/src/styles/components/service.css [hunk 1]
```diff
@@ -24,5 +29,22 @@
...
 
 .service-description-container {
-  margin-left: 20px;
+  margin-left: 50px;
+}
+
+.service-list-wrapper {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(3, 24px);
```
Comment (hungdo-agilityio) [inline practice-one/src/styles/components/service.css]: Do we need to define rows?

File: practice-one/src/components/service.html [hunk 2]
```diff
@@ -3,5 +3,36 @@
   <div class="service-description-container">
     <p class="service-header">We are <span class="primary-color">more</span> than <span class="accent-color">multiple</span> service</p>
     <p class="service-description">This is a type of resturent which typically serves food and drink, in addition to light refreshments such as baked goods or snacks. The term comes frome the rench word meaning food.</p>
+    <div class="service-list-wrapper">
+      <div class="service">
+        <img class="service-icon" src="./assets/images/order-online.svg"/>
+        Online order
+      </div>
```
Comment (hungdo-agilityio) [inline practice-one/src/components/service.html]: Recheck the html tag

Comment (CBinh710) [reply practice-one/src/components/service.html]: Thanks i understood

File: practice-one/src/styles/components/service.css (hunk 1 above)
Comment (CBinh710) [reply practice-one/src/styles/components/service.css]: Dont need to define it, thanks for your comment

**Review by hungdo-agilityio — APPROVED** [review]

**Pull request merged**